src/utils.ts

```diff
@@ -22,6 +23,25 @@ export async function validate(tokenList: JsonableTokenList) {
   return false;
 }
 
+export async function validateTokensObject(tokens: Token[]) {
+  if (MainnetTokenList.tokens.length) {
```

Comment:
I think this function should be removed and instead we can use jest expect to test if objects are equal. There is a lot of option to compare objects in jest itself, so this function feels not needed.